itwin-platform-access/imodels-access-frontend/package.json

```diff
@@ -35,15 +35,15 @@
     "@itwin/imodels-client-management": "workspace:*"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.6.0",
-    "@itwin/core-bentley": "^3.6.0",
-    "@itwin/core-common": "^3.6.0",
-    "@itwin/core-frontend": "^3.6.0",
-    "@itwin/core-geometry": "^3.6.0",
-    "@itwin/core-orbitgt": "^3.6.0",
-    "@itwin/core-quantity": "^3.6.0",
+    "@itwin/appui-abstract": "^3.6.0 || ^4.0.0",
```

Comment:
change dev & direct deps to use `^4.0.0`
change peer deps to `^3.6.0 || ^4.0.0`

Reply:
changed